### PR TITLE
Added kubernetes subprojects

### PR DESCRIPTION
**Pull request by eddie-knight**
ref: https://docs.google.com/spreadsheets/d/1Y9hNtp6NaEtp_B0ccwMI06YBdENz3guh93MlPHwdPXM/edit#gid=0

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
https://github.com/cncf/clomonitor/pull/1378

File: data/cncf.yaml [hunk 1]
```diff
@@ -3426,3 +3426,73 @@
...
+    - name: kompose
+      url: https://github.com/kubernetes/kompose
+      check_sets:
+        - community
+        - code
+- name: karpenter
+  display_name: Karpenter
+  description: Karpenter is a Kubernetes Node Autoscaler built for flexibility, performance, and simplicity.
+  category: cluster orchestration and management
+  logo_url: https://raw.githubusercontent.com/kubernetes/kubernetes/master/logo/logo.png
```
Comment (tegioz) [inline data/cncf.yaml:3475]: Can you use the SVG version of the Kubernetes logo?

`https://raw.githubusercontent.com/kubernetes/kubernetes/master/logo/logo.png`

**Review by tegioz — CHANGES_REQUESTED** [review]
Thanks @eddie-knight!

**Comment by eddie-knight** [comment]
@tegioz please take another look, I'm pretty sure I've applied all the requested changes

**Comment by tegioz** [comment]
Thanks @eddie-knight, it looks great 👍

Would you also like to add the `cloud-provider-aws` (#1384) and `gateway-api` (#1381) subprojects to this PR? They kind of have to fix the same details and we could leave them all ready now 🙂

**Comment by eddie-knight** [comment]
I went ahead and pulled in the commits from the other PRs and rebased them into a single history here

#1378
#1379 
#1380
#1381
#1382
#1384
#1386

**Review by tegioz — APPROVED** [review]
Thanks! 🥇

**Comment by afbjorklund** [comment]
Should probably add both `k8s.io/minikube` and `sigs.k8s.io/kind`, since they share some of the container code?

https://github.com/kubernetes/minikube

https://github.com/kubernetes-sigs/kind

The minikube project is older than the creation of the kubernetes-sigs organisation, so it still lives in kubernetes.

**Pull request merged**